File: Data Structures & Algorithms/anagram-groups/submission-0.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& strs) {
        unordered_map<string, vector<string>> my_map;
        for(int i = 0; i < strs.size(); i++){
            string value = strs[i];
            sort(strs[i].begin(), strs[i].end());
            my_map[strs[i]].push_back(value);
        }

        vector<vector<string>> output;
        for (const auto& pair : my_map) {
            vector<string> temp;
            for (const auto& value : pair.second) {
                temp.push_back(value);
            }
            output.push_back(temp);
        }
        return output;
    }
};

File: Data Structures & Algorithms/is-palindrome/submission-0.py
class Solution:
    def isPalindrome(self, s: str) -> bool:

        i, j = 0, len(s)-1

        while(i<=j):
            if(s[i] != s[j]):
                return false;
            i += 1;
            j -= 1;

        return true;

        